packages/web-components/fast-foundation/src/slider/slider.template.ts

```diff
@@ -14,7 +14,7 @@ export function sliderTemplate<T extends FASTSlider>(
         <template
             role="slider"
             class="${x => (x.readOnly ? "readonly" : "")}
-        ${x => x.orientation || Orientation.horizontal}"
+                ${x => x.orientation || Orientation.horizontal}"
```

Comment:
I think both of these can be removed in favor of the attributes. Classes on the host are an anti-pattern we want to avoid wherever possible. We can take as a follow-up as needed.

Reply:
This sounds good to me. I did this in a couple of places, but I probably missed a few, and I wasn't entirely sure if we landed on that convention or not.

Reply:
I went back and removed any classes which were also attributes on all components.